src/components/UI/common/Footer.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Center, Divider, Flex, Link, Stack } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import efLogoSVG from '../../../public/images/ef-logo.svg';
```

Comment:
Minor nitpick but I do like to see consistent order of imports, e.g. something like:
1. libraries (react, chakra-ui)
2. components
3. images

Nothing blocking this PR but something to consider moving forward.

Reply:
You may have already thought this through :) LMK if you have an opinion.

Reply:
I'm currently using the [`organizeImports` ](https://eshlox.net/2019/12/02/vscode-automatically-organize-typescript-imports/) from vscode which sorts them alphabetically. I'll check if I can add an eslint rule to enforce that pattern 👍.

Reply:
Oh, nice! Didn't know about that - I like it 👍